.github/workflows/tests.yml

```diff
@@ -353,8 +342,5 @@ jobs:
       - name: Remark
         run: yarn lint:markdown
 
-      - name: Example schemas
-        run: yarn lint:examples
```

Comment:
This is unnecessary now since it happens in our `prepare` script (which happens after install)